README.md

```diff
@@ -16,3 +18,30 @@ The installation of the package is standard: for details, see GAP reference
 manual, chapter "GAP Packages" (and if you don't have write access to the
 Gap installation, type in GAP "?GAP root directory" to see another relevant
 part of the manual).
+
+## Documentation
+
+Full information and documentation can be found in the manual, available
+as PDF `doc/manual.pdf` or as HTML `htm/chapters.htm`, or on the package
+homepage at
+
+  <http://gap-packages.github.io/OpenMath/>
+
+
+## Bug reports and feature requests
+
+Please submit bug reports and feature requests via our GitHub issue tracker:
+
+  <https://github.com/gap-packages/OpenMath/issues>
+
+
+License
+=======
+
+datastructures is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2 of the License, or (at your
+option) any later version.
+
+For details see the files COPYRIGHT.md and LICENSE.
```

Comment:
I think this copyright info is provided elsewhere. Is there a recommendation for all packages to have now one more *.md file which COPYRIGHT.md ?